concepts/booleans/introduction.md

```diff
@@ -1,3 +1,8 @@
 # Introduction
 
-TODO
+Booleans in Rust are represented by the `bool` type, which values can be either `true` or `false`.
+
+Rust supports six comparison operators: `==` (Equal), `!=` (Not equal), `>`	(Greater than), `<` (Less than), `>=`
```

Comment:
Probably worth adding a note about why these operators are relevant here: they take two non-boolean values and produce a boolean.